sdk/deploy.js

```diff
@@ -22,6 +22,7 @@ const deployCode = async function deployCode(deployURLBase, apiKey, tarPath) {
     headers: {
       'Content-Type': 'application/gzip',
       'X-Binaris-Api-Key': apiKey,
+      'X-Binaris-CLI-Version': version,
```

Comment:
isn't this the sdk version?

Reply:
I'll change it to `X-Binaris-Client-Version`.

Reply:
I'd add the header in `loggedRequest`, simply check if `headers` are passed in and add the version header to the ones passed in. (see comment below)